examples/external-security/gradle-dev.properties

```diff
@@ -0,0 +1,8 @@
+mlHost=dev-host
```

Comment:
I don't think it's necessary to show different environment properties files in action - I think it distracts from the purpose of showing how to configure external security. I think just use "gradle.properties", no need for the different environment files.